### PR TITLE
feat(pagination): offer compact variation

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/2110

### CODE REVIEW

File: src/patternfly/components/Pagination/pagination.scss [hunk 1]
```diff
@@ -12,6 +13,7 @@
   --pf-c-pagination__nav--c-button--PaddingLeft: var(--pf-global--spacer--sm);
   --pf-c-pagination__nav--c-button--PaddingRight: var(--pf-global--spacer--sm);
   --pf-c-pagination__nav--c-button--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-pagination--m-compact__nav--c-button--MarginLeft: var(--pf-global--FontSize--md);
```
Comment (christiemolloy) [inline src/patternfly/components/Pagination/pagination.scss]: Why does this use a global FontSize var for the Margin?

**Comment by christiemolloy** [comment]
@mcoker was the plan to update this in any spots or leave that to another issue? It would be good to see an example of this in the table/data list demos to test how it looks

File: src/patternfly/components/Pagination/pagination.scss (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/Pagination/pagination.scss]: Doh! Because I copied and pasted the wrong thing :) Updated.

**Comment by gdoyle1** [comment]
@mcoker Hmm for some reason the compact pagination example looks exactly like the normal pagination. Not sure what's going on! This is what I'm seeing
Screen Shot 2019-09-18 at 10 41 25 PM

**Comment by mcoker** [comment]
@gdoyle1 oh sorry about that, the PR build site has a bug where if you click on an example that opens in a full page, it takes you to a page with all of the examples for that component. To see the compact navigation, just scroll to the very bottom since it's the last example for the component.

**Comment by mcoker** [comment]
@mcarrano there are no other differences between the compact and full pagination other than the spacing in the top level pagination, illustrated in the compact example. The expanded menu, disabled buttons, etc would look the same as they do in the full pagination, which there are already examples of. But if you think it's helpful to include those for reference, I'm happy to create a follow up issue and add them.

**Comment by gdoyle1** [comment]
> @mcarrano there are no other differences between the compact and full pagination other than the spacing in the top level pagination, illustrated in the compact example. The expanded menu, disabled buttons, etc would look the same as they do in the full pagination, which there are already examples of. But if you think it's helpful to include those for reference, I'm happy to create a follow up issue and add them.

I agree with you @mcoker on this one!

**Pull request merged**